src/identifiers.rs

```diff
@@ -218,16 +218,10 @@ impl RoomId {
         Ok(Self::from(ruma::RoomId::parse(id)?))
     }
 
-    /// Returns the user's localpart.
-    #[wasm_bindgen(getter)]
-    pub fn localpart(&self) -> String {
-        self.inner.localpart().to_owned()
-    }
```

Comment:
removing this seems like a breaking change. Can we not maintain its behaviour somehow?

Reply:
It's not used anywhere yet, so I was more keen to drop it and not having to maintain it?

I tried to use same reasoning that on the ruma change log

> Room IDs being splittable into localpart and servername does not have
much inherent value and there are proposals like MSC4051¹ that propose
changing the format. Relaxing the rules makes Ruma forwards-compatible
with those proposals. The server_name accessor is kept because it is
used by at least one downstream, but is updated to return an `Option`.

Reply:
ok well, we better document it in the changelog as a breaking change.

Reply:
Good idea, I added a draft changelog.md for next version.

Reply:
As per our discussion I also removed RoomId.serverName